modules/collectd/manifests/plugin/docker.pp

```diff
@@ -2,7 +2,10 @@
 #
 # Collectd plugin for docker
 #
-class collectd::plugin::docker {
+class collectd::plugin::docker(
+  $repo = undef,
+  $commit = undef,
```

Comment:
I think you should set these to the values from hieradata and then delete them from hieradata - we shouldn't need it there since this will be very static.

Reply:
The test actually worked with hieradata. I am going to leave it there.